build/ps-entrypoint.sh

```diff
@@ -180,9 +180,17 @@ create_default_cnf() {
 }
 
 load_group_replication_plugin() {
+	POD_IP=$(hostname -I | awk '{print $1}')
+
 	sed -i "/\[mysqld\]/a plugin_load_add=group_replication.so" $CFG
 	sed -i "/\[mysqld\]/a group_replication_bootstrap_group=OFF" $CFG
 	sed -i "/\[mysqld\]/a group_replication_start_on_boot=OFF" $CFG
+	sed -i "/\[mysqld\]/a group_replication_consistency=EVENTUAL" $CFG
+	sed -i "/\[mysqld\]/a group_replication_exit_state_action=READ_ONLY" $CFG
+	sed -i "/\[mysqld\]/a group_replication_communication_stack=XCOM" $CFG
```

Comment:
XCOM is a legacy protocol, can we use MySQL here?

Reply:
we can but is less tested in the gr environment and you really need to be careful about fulfill all these https://dev.mysql.com/doc/refman/8.0/en/group-replication-connection-security.html